mydumper.c

```diff
@@ -1875,7 +1888,9 @@ GString * get_insertable_fields(MYSQL *conn, char *database, char *table){
 			g_string_append(field_list, ",");
 		}
 
-		g_string_append_len(field_list, bind.buffer, *bind.length);
+		GString *original = g_string_new_len(bind.buffer, *bind.length);
+        append_escaped_identifier(field_list, original->str);
```

Comment:
most code in this repo uses `tab` as indent, better to be unified

Reply:
done in https://github.com/pingcap/mydumper/pull/28/commits/520f0789472d0e0fb72405e29bb8810e5e0c0884.